src/lib/Input.svelte

```diff
@@ -9,23 +9,19 @@
 
 <script lang="ts">
     import { createEventDispatcher } from "svelte";
+    import { resetStore } from "./messageStore";
 
     const dispatch = createEventDispatcher<Events>();
 
     let innerText = "";
     let messageReceived = true;
 
-    /**
-     * Removes all white spaces from the innerText.
-     *
-     * @returns {void}
-     */
-    function getRidOfWhiteSpacesInInnerText(): void {
-        innerText = innerText.replace(/\s/g, "");
-    }
+    const getRidOfWhiteSpacesBeforeAndAfter = () => {
```

Comment:
Noice! There's no reason to be this obsessive about saving space, but because I sometimes love trying to limit the number of symbols I use, just watch to mention that this could effectively be implement as a one liner (though the svelte linter might still push into onto two lines) as:

```ts
    const getRidOfWhiteSpacesBeforeAndAfter = () => (innerText = innerText.trim())
```

Reply:
Oh yeah. The thing is the function before was actually taking out of every single white space in the string so if you were to type `hello world` it turns it into `helloworld` so that's why I just rewrote it using the `trim` function.